department-of-veterans-affairs/caseflow
# Add keyboard shortcuts in reader for categories

Resolves #8519 

### Description
Adds keyboard shortcuts in Reader for adding and removing category labels Procedural (Alt+Shift+P), Medical (Alt+Shift+M), and Other Evidence (Alt+Shift+O).

### Acceptance Criteria
- [ ] New shortcuts work
- [ ] Existing shortcuts continue to work

### Testing Plan
1. Go to reader view (ex: http://localhost:3000/reader/appeal/2226048/documents/21)
2. Send the new keyboard shortcuts and ensure the categories are toggled

### User Facing Changes


<img width="800" alt="image" src="https://user-images.githubusercontent.com/2308626/113866543-73880e00-977b-11eb-88df-6ed9f2f82008.png">
